client/src/pages/signin/Signin.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from "react"
+import './Signin.scss'
+import { Link } from 'react-router-dom'
+
+import TextField from '@material-ui/core/TextField';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+
+class signin extends Component{
+
+    constructor(){
+        super()
+
+        this.state={
+            login: {
+                email: '',
+                password: ''
+            },
+            passLength: 0,
+            snackbaropen: false,
+            snackbarmsg: ''
+        }
+    }
+
+    //handles the snackbar warning message
+    snackbarClose = (event) => {
+        this.setState({
+            snackbaropen: false
+        })
+    }
+
+    //handle updating the state with the values of the input fields
+    updateText(event){
```

Comment:
make this into es6 syntax. that way, you probably won't have to bind `this` later in line 80

Reply:
thank you for highlighting this, es6 syntax changes made